modules/upgrade/pages/k-rolling-upgrade.adoc

```diff
@@ -3,56 +3,41 @@
 :page-context-links: [{"name": "Linux", "to": "upgrade:rolling-upgrade.adoc" },{"name": "Kubernetes", "to": "upgrade:k-rolling-upgrade.adoc" } ]
 :page-aliases: manage:kubernetes/rolling-upgrade.adoc
 
-To benefit from Redpanda's new features and enhancements, use rolling upgrades to upgrade to the latest version. New features are available after all brokers (Pods) in the cluster are upgraded and restarted.
+To benefit from Redpanda's new features and enhancements, upgrade to the latest version. New features are available after all brokers in the cluster are upgraded and restarted.
 
-Redpanda platform version numbers follow the convention AB.C.D, where AB is the two digit year, C is the feature release, and D is the patch release. For example, version 22.3.1 indicates the first patch release on the third feature release of the year 2022. Patch releases include bug fixes and minor improvements, with no change to user-facing behavior. New and enhanced features are documented with each feature release.
+Redpanda platform version numbers follow the convention AB.C.D, where AB is the two digit year, C is the feature release, and D is the patch release. For example, version 22.3.1 indicates the first patch release on the third feature release of the year 2022. Patch releases include bug fixes and minor improvements, with no change to user-facing behavior. New and enhanced features are documented with each feature release. You can find a list of all releases on https://github.com/redpanda-data/redpanda/releases[GitHub^].
 
 include::partial$rolling-upgrades/important-upgrade-note.adoc[]
```

Comment:
These all look like limitations...have you considered a heading "Limitations" and placing the content under it?

Reply:
@Feediver1 I added a limitation section. Please could you review that section.

Reply:
https://github.com/redpanda-data/docs/pull/110/files#diff-df80463050405ef02d895b5e2dd4f7f063957887c44793d11872c275080161bb

Reply:
cc @micheleRP since this is single-sourced into the Linux upgrade guide too

Reply:
oh, lovely! This is much nicer.